src/annex-b-fns/global-existing-global-init.case

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+desc: >
+  Variable binding is left in place by legacy function hoisting.
+  CreateGlobalVariableBinding leaves the binding as non-enumerable even
+  if it has the chance to change it to be enumerable.
+template: global
+info: |
+    B.3.3.3 Changes to GlobalDeclarationInstantiation
+
+    [...]
+    Perform ? varEnvRec.CreateGlobalVarBinding(F, true).
+    [...]
+
+includes: [fnGlobalObject.js, propertyHelper.js]
+---*/
+
+//- setup
+Object.defineProperty(fnGlobalObject(), 'f', {
+  value: 'x',
+  enumerable: true,
+  writable: true,
+  configurable: false
+});
+
+$.evalScript(`
```

Comment:
Oh oops, will fix.

We need to be eval'ing another script; this has different behavior. I'm getting at the global case in this test, not the eval behavior. test262 is actually missing coverage of other properties of Annex B 3.3 extensions to GlobalDeclarationInstantiation.